packages/nimble-components/src/tabs/specs/scrollable-tabs.md

```diff
@@ -0,0 +1,110 @@
+# Scrollable tabs
+
+## Problem Statement
+
+There are some use-cases that call for many tabs, and in cases where the width of all of the tabs exceed the screen space, they will bleed off screen (and potentially make the page scrollable to see the other tabs). To prevent this, we will introduce "scrollable" tabs, which will enforce the tabs to be constrained to the given space of the `Tabs` component and offering buttons to allow a user to navigate through the whole set of tabs.
+
+### Out of scope of this HLD
+
+Any decision/API around limiting the width of a particular tab. This is ultimately an orthogonal concern to the UX this HLD is presenting.
+
+## Links To Relevant Work Items and Reference Material
+
+-   [Issue 1509: Scrollable Tabs](https://github.com/ni/nimble/issues/1509)
+-   [Figma design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=11133-57220&node-type=instance&t=DlnrdRo7KWYlheFz-0)
+
+## Implementation / Design
+
+![Scrollable Tabs](./spec-images/scrollable-tabs.png)
+
+The following decisions were agreed to in the comments section in the linked issue above:
+
+-   No scrollbar will be shown
+-   Keyboard navigation will work as it does today, but when a tab that isn't fully in view gets focus it should be scrolled into view
+-   When a scroll button is pressed we will scroll either a full "page width" or the remainder of the available space, and tab focus is not changed.
+
+Additionally, the mouse wheel should allow scrolling through the tabs, but instead of scrolling a page at a time, a single scroll "tick" will scroll by one tab.
```

Comment:
Is it a requirement to support pan scrolling on mobile? Do we need to do anything to support this or do we get it for free?

Reply:
I imagine this is _not_ free. Pan scrolling is usually accomplished by a "click-drag" operation, the behavior of which will not be provided by the tabs. Is it worth calling this out as out-of-scope?